python/tvm/contrib/download.py

```diff
@@ -41,8 +41,8 @@ def download(url, path, overwrite=False, size_compare=False, verbose=1, retries=
     size_compare : bool, optional
         Whether to do size compare to check downloaded file.
 
-    verbose: int, optional
-        Verbose level
+    verbose: bool, optional
+        If status messages should be printed.
```

Comment:
```suggestion
        If status messages should be printed. Defaults to false.
```

Reply:
I like it, and I would add it if not for the change/removal of `verbose` following your suggestion to use `logging`.